### PR TITLE
User admin bourbon styling.

### LINKED ISSUE #56: Apply bourbon styling to User admin views

### PR DESCRIPTION
**Pull request by josephbridgwaterrowe**
Update the user admin to use bourbon styling for consistency.
 - Add user entry for validation when editing user details.
 - Closes #56.

### CODE REVIEW

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -14,15 +15,15 @@ def index
       @users = User.all
     end
 
-    def show
-    end
+    def show; end
 
     def update
-      if @user.update_attributes(user_params)
+      return render :edit unless @user_entry.valid?
```
Comment (michaelachrisco) [inline app/controllers/admin/users_controller.rb]: Kind of confusing. I would change to:
```Ruby
return render :edit if !@user_entry.valid?
```

**Comment by josephbridgwaterrowe** [comment]
You find the unless the user is valid confusing?

I think it reads more naturally. :)

File: app/controllers/admin/users_controller.rb (hunk 1 above)
Comment (michaelachrisco) [inline app/controllers/admin/users_controller.rb]: in english:
1. ```Return reder Edit, unless user_entry is valid.```
2. ```Return render Edit, if user_entry is not valid.```
but I guess one could argue that 2. is also:
2a. ```Return render Edit, if not user_entry is valid.```

I guess unless is just as good in this situation.

**Comment by michaelachrisco** [comment]
That is the only issue (non issue?) that I can see. Merging.

**Pull request merged**